<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="leftTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="jobsTab">
        <property name="minimumSize">
         <size>
          <width>123</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <attribute name="title">
         <string>Jobs</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QFrame" name="jobListFrame">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="newJobButton">
                   <property name="text">
                    <string>New</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="newJobLineEdit"/>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QListWidget" name="jobListWidget"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="currentJobWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="loadFileButton">
                   <property name="text">
                    <string>Load</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="loadFileLineEdit"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QListWidget" name="currentJobListWidget"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="setsTab">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <attribute name="title">
         <string>Sets</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>New</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="setTableWidget"/>
             </item>
            </layout>
           </widget>
           <widget class="QTextEdit" name="setTextEdit"/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pointsTab">
        <attribute name="title">
         <string>Points</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QTableView" name="pointsTableView"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QQuickWidget" name="quickWidget">
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
      <widget class="QToolBox" name="rightToolBox">
       <widget class="QWidget" name="rightToolBoxPage1" native="true">
        <attribute name="label">
         <string>COGO</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="rightToolBoxPage2" native="true">
        <attribute name="label">
         <string>Point Manipulation</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="rightToolBoxPage3" native="true">
        <attribute name="label">
         <string>Point/Observation Editor</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>newJobButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newJobSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newJobLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>jobNameSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFileSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFileLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>dataFilenameSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newSetSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadSetSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveSetSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newJobSlot()</slot>
  <slot>jobNameSlot()</slot>
  <slot>loadDataSlot()</slot>
  <slot>dataFilenameSlot()</slot>
  <slot>newSetSlot()</slot>
  <slot>loadSetSlot()</slot>
  <slot>saveSetSlot()</slot>
 </slots>
</ui>
